src/core/federation.ts

```diff
@@ -281,6 +287,123 @@ function isANodeJSCodedError(e: Error): e is NodeJSCodedError {
   return typeof (e as NodeJSCodedError).code === "string";
 }
 
+/**
+ * A network sender is a reactor containing a portAbsentReaction.
+ */
+export class NetworkSender extends Reactor {
+  /**
+   * The last reaction of a NetworkSender reactor is 'portAbsentReactor'.
+   * @returns portAbsentReactor of this NetworkSender reactor
+   */
+  public getLastReactioOrMutation(): Reaction<Variable[]> | undefined {
```

Comment:
The name of this method has a typo. But aside from that, why is it needed? This looks like a "no-op" override...

Reply:
Because `_getLastReactionOrMutation` in `Reactor` is `protected` but for `NetworkSender` it needs to be exposed.

https://github.com/lf-lang/reactor-ts/blob/08fa08468d2c45714cd8e1fe6bd24f83edadda89/src/core/federation.ts#L1475-L1482

Reply:
Yes, `NetworkSedner` does not extend `FederatedApp`, and `FederatedApp` has to access the `NetworkSender`'s last reaction because it's a port absent reaction.

Reply:
OK, I just renamed the method to reflect that it's just meant to retrieve the "port absent" reaction.